motr/idx_dix.c

```diff
@@ -577,10 +577,13 @@ static void dix_build(const struct m0_op_idx *oi,
 		 * argument.
 		 */
 		out->dd_layout.dl_type = DIX_LTYPE_DESCR;
+
 		m0_dix_ldesc_init(&out->dd_layout.u.dl_desc,
 				  &(struct m0_ext) { .e_start = 0,
 				  .e_end = IMASK_INF }, 1, HASH_FNC_CITY,
-				  &idx->in_attr.idx_pver);
+				  (oi->oi_flags & M0_ENF_META) ?
```

Comment:
[Style]
These braces can be removed.
(oi->oi_flags & M0_ENF_META)

Reply:
It is added to avoid codacy issue - operator precedence between "&" and "?"